src/jvlink/bridge.py

```
@@ -109,35 +177,323 @@ def __init__(
-            stdin=subprocess.PIPE,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            text=True,
-            encoding="utf-8",
-            bufsize=1,
-        )
+        cmd = self._build_command()
+        stderr_file = self._open_stderr_file()
+
+        try:
+            self._process = subprocess.Popen(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)